clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -67,6 +70,31 @@ export default async function handler(
     }));
   }
 
+  // Check if a geolocation was provided via headers, query param, or obtainable via a geolocation URL;
+  // if so, inject into the bundle, along with privacy experience
+  const geolocation = await lookupGeolocation(req, environment.settings);
+  let experience;
+  if (geolocation) {
+    const fidesRegionString = constructFidesRegionString(geolocation);
+    if (fidesRegionString && environment.settings.IS_OVERLAY_ENABLED) {
+      let fidesUserDeviceId = null;
+      if (Object.keys(req.cookies).length) {
+        const fidesCookie = req.cookies[CONSENT_COOKIE_NAME];
+        if (fidesCookie) {
+          fidesUserDeviceId =
+            JSON.parse(fidesCookie)?.identity?.fides_user_device_id;
+        }
+      }
+      console.log("Fetching relevant experiences from server-side...");
```

Comment:
do we want this? it might spam the logs with how often this endpoint can get called. or maybe we also want to only log depending on `settings.DEBUG`

Reply:
good point, I'll wrap this based on `settings.DEBUG`